assimilation_code/modules/assimilation/filter_mod.f90

```diff
@@ -1761,6 +1765,56 @@ end subroutine obs_space_diagnostics
 
 !-------------------------------------------------------------------------
 
+subroutine obs_space_sync_QCs(obs_fwd_op_ens_handle,  &
+   seq, keys, num_obs_in_set, OBS_GLOBAL_QC_COPY, DART_qc_index)
+
+! If QCs were updated in the assimilation loop but posterior forward operators
+! are not being computed, collect any updated QCs into the output obs_seq.
```

Comment:
remove this comment too,  
I don't think the function should explain the calling code, and also the calling code has a comment explaining why at line 1054
```suggestion
```